ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -271,6 +271,7 @@ internal class PaywallViewModelImpl(
             validatedPaywallData = displayablePaywall,
             template = template,
             shouldDisplayDismissButton = options.shouldDisplayDismissButton,
+            actionInProgress = actionInProgress.value,
```

Comment:
I added this here, then realized that we need to update the state in `verifyNoActionInProgressOrStartAction()`, where `actionInProgress` is updated. I am not sure there's a more efficient way of doing it. The alternative would be to update the button wherever the `PaywallListener` is implemented

What do you think @tonidero

Reply:
Hmm we're already exposing `actionInProgress` from the `PaywallViewModel` as a different `State`. Couldn't we just use that in the `InternalPaywall` directly? I believe in that case this wiring won't be necessary? Do lmk if I'm missing something!

Reply:
oh that's right, I missed we were exposing that as a state already